src/qt/guiutil.cpp

```diff
@@ -781,12 +779,12 @@ void setClipboard(const QString& str)
 
 fs::path qstringToBoostPath(const QString &path)
 {
-    return fs::path(path.toStdString(), utf8);
+    return fs::path(path.toStdString());
```

Comment:
doesn't this break support for non-UTF8 path locales on UNIX?

Reply:
that utf8 is unused on other platform than Windows
https://github.com/boostorg/filesystem/blob/16821f7903f02d0bb6bb2ef2503ea41559c6bdf1/include/boost/filesystem/path.hpp#L411-455

Reply:
strange…

Reply:
@laanwj You only mention about Windows in ＃3935